### PR TITLE
Frozen editor assets only back

### PR DESCRIPTION
**Pull request by AleGoiko**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -127,14 +127,67 @@ def insert_google_maps(query_string)
...
   end
 
   def stylesheet_link_tag *sources
     raise_on_asset_absence sources
+
     super *sources
   end
 
+  def editor_assets_path
+    file = File.read("./config/editor_assets_version.json")
```
Comment (javitonino) [inline app/helpers/application_helper.rb]: We probably want this to be memoized, so we don't read the file all the time. We could also move it to `Carto::Configuration` (like `frontend_version`).

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -127,14 +127,67 @@ def insert_google_maps(query_string)
...
+      tag_options = {
+        "src" => href
+      }.merge!(options)
+      content_tag("script".freeze, "", tag_options)
+    }.join("\n").html_safe
+
+    sources_tags
+  end
+
+  def editor_favicon_link_tag(source = "favicon.ico", options = {})
```
Comment (javitonino) [inline app/helpers/application_helper.rb]: Is the favicon different for Builder and Editor?

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -127,14 +127,67 @@ def insert_google_maps(query_string)
...
+  end
+
+  def editor_stylesheet_link_tag(*sources)
+    raise_on_asset_absence sources
+
+    options = sources.extract_options!.stringify_keys
+    path = editor_assets_path
+
+    sources_tags = sources.uniq.map { |source|
+      href = "#{path}/stylesheets/#{source}"
```
Comment (javitonino) [inline app/helpers/application_helper.rb]: This doesn't take into account the `assets_host` among other things. Why not call the Rails function instead?

Comment (AleGoiko) [reply app/helpers/application_helper.rb]: :thinking:  but its a relative path as far as I could see

Comment (javitonino) [reply app/helpers/application_helper.rb]: Look at the HTML in production, we use absolute paths to reference the CDN. In a local install, yes, paths are relative. All of this is internally handled by Rails (`javascript_include_path`)

**Comment by AleGoiko** [comment]
retest this please

**Comment by AleGoiko** [comment]
merged into `frozen-editor-assets`

**Pull request merged**